### DISCUSSION TITLE
API for billing ussage

### DISCUSSION THREAD

**Discussion by BenBroide** [vercel/community#715] [Help] [answered]
Is there a way to fetch the billing ussage via API for automating the monitoring of ussage to reach high bills?
Thanks

**Comment by BossBele** [comment, marked as answer]
Currently, Vercel does not have an API to get usage and billing stats/data. Information about usage and billing can be found on the Dashboard.

**Comment by brianmichel** [comment]
Any possibility on getting an API that allows us to query billing or usage to help teams get better visibility of bills across cloud platforms? I see that this is answered, but wanted to see if we can file a feature request to get this.
